config/deploy.rb

```diff
@@ -6,6 +6,7 @@
 set :rvm_ruby_version, '2.5.1'
 set :assets_roles, [:web, :app]
 set :npm_flags, '--silent --no-progress' # Not use --production to install devDependencies
+set :whenever_command, ->{ [:bundle, :exec, :whenever] }
```

Comment:
`set :whenever_command, "bundle exec whenever"` でいいと思うんですけど、lambda使ってるの何ででしょう？
https://github.com/javan/whenever#capistrano-integration

Reply:
ここを参考にしてました

https://github.com/javan/whenever/blob/master/lib/whenever/capistrano/v3/tasks/whenever.rake#L46

Reply:
どうせ内部的にStringに変換して二度手間ですし、最初からStringで
```suggestion
set :whenever_command, "bundle exec whenever"
```